### PR TITLE
chore: bump min xcode version to 11

### PR DESCRIPTION
**Pull request by KazuCocoa**
Since Xcode 11, built WDA bundle id changes to `.xctrunner` postfix instead of `com.apple.test.XXXXX`.
We've already explicitly announced to support latest 2 versions, but let's drop 10 explicitly to not consider the older bundle id era.

---

this version will bump major version.
Then, will update https://github.com/appium/appium-xcuitest-driver

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
Do we need to mention such changes in README?

**Comment by mykola-mokhnach** [comment]
Also, e2e tests are still not happy. I assume some changes did happen to the destination selenium web page that they use for the validation.

UPD: I have checked the web site used in that test and it does not seem to exist anymore. So I've disabled the test for now in my other PR

**Comment by KazuCocoa** [comment]
I thought to update in xcuitest-driver and appium repo (so far, only these readme address such versions), but let me address versions in this readme as well

**Review by mykola-mokhnach — APPROVED** [review]

File: .azure-pipelines.yml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+  MIN_VM_IMAGE: macOS-10.15
+  MIN_XCODE_VERSION: 11.2.1
   # Xcode 10.0 did not have Apple TV
-  MIN_TV_XCODE_VERSION: 10.2
-  MIN_TV_PLATFORM_VERSION: 12.2
-  MIN_TV_DEVICE_NAME: Apple TV
-  MIN_PLATFORM_VERSION: 12.2
-  MIN_IPHONE_DEVICE_NAME: iPhone X
-  MIN_IPAD_DEVICE_NAME: iPad Air 2
+  MIN_TV_XCODE_VERSION: 11.2.1
```
Comment (KazuCocoa) [inline .azure-pipelines.yml:6]: this version is the available Xcode 11 min version on Azure

**Pull request merged**